fakebook/src/app/services/auth.service.ts

```diff
@@ -14,17 +14,31 @@ export class AuthService {
   isAuthenticated: boolean | unknown;
 
   constructor(public oktaAuth: OktaAuthService, public router: Router) {
+    this.updateAuthState(false);
+    this.oktaAuth.$authenticationState.subscribe((isAuthenticated) =>
+    this.updateAuthState(isAuthenticated)
+    );
   }
 
   updateAuthState(isAuthenticated: boolean): void {
+    this.isAuthenticated = isAuthenticated;
+    if (isAuthenticated) {
+      this.oktaAuth.getUser().then(console.log);
+    }
   }
 
   subscribeAuthStateChange(updateFn: (authState: boolean) => void): void {
+    this.oktaAuth.$authenticationState.subscribe((authState) => updateFn(authState));
   }
 
   login(): void {
+    this.oktaAuth.signInWithRedirect({
+      originalUri: 'newsfeed'
+    });
```

Comment:
why should this service know or care what route to redirect to? not it's responsibility.

we can compromise process and make non major corrections to the tests in PRs like this too i guess